frontend/src/app/application-forms/validators/error-message.component.spec.ts

```diff
@@ -51,7 +51,7 @@ describe('ErrorMessageComponent', () => {
       min: ['', Validators.min(1)]
     });
     form.get('emailAddress').setValue('test');
-    expect(component.parseErrors(form.get('emailAddress').errors)).toEqual('Test requires a valid email address. ');
+    expect(component.parseErrors(form.get('emailAddress').errors)).toEqual('Test requires a valid email address. I.E. you@email.com ');
```

Comment:
@jdamis4XD @DawnDesigner @lisaredux Is this change to the invalid email error message ok?

Reply:
Thanks Dave. I'm not sure I see the update. 

I don't have a problem with the actual message — just that it comes too early before someone has clicked into that text input field. Ideally should show when someone clicks out of that field.

<img width="465" alt="Screenshot 2019-05-07 09 52 00" src="https://user-images.githubusercontent.com/26801590/57304823-06aafb80-70ae-11e9-9af4-5265739f3695.png">

I defer to Amber about the content of the message itself.

Reply:
@lisaredux the email confirmation validation is a different issue (#489 ) Here, Mike made a change to the **message** the user sees when an invalid email address is entered and I just want to make sure that it is ok with y'all.

## Before
`Email requires a valid email address.`

## After
`Email requires a valid email address. I.E. you@email.com`

Reply:
Thanks @davemcorwin  — I see now. Thanks for clarifying. 
I recommend: "Requires a valid email address— i.e. you@email.com"
Though I'm going to defer to @ASprinkle on the wording and standards.

Reply:
Yes, the timing piece is a different story.  Agreed, "Requires a valid email address" when an invalid email address is entered.  There should be a separate error note for when the email verification entered does not match... Correct?

Reply:
@ASprinkle Yes, and there is. No changes are made here to the email confirmation validation error.

Reply:
@lisaredux sounds like we want it to read: Requires a valid email address— i.e. you@email.com
as opposed to: Email requires a valid email address. I.E. you@email.com.

I'll get that change implemented before the merge